client/js/screens/SubmitReview/SubmitReview.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import {View, Image, Text, ScrollView, AsyncStorage} from 'react-native';
+import ButtonDefault from '../../components/ButtonDefault';
+import ReviewScore from '../../components/ReviewScore';
+import {calculateRatingScore} from '../../lib/calculateRatingScore';
+import styles from './styles';
+import SafeAreaView from 'react-native-safe-area-view';
+import InputDefaultField from '../../components/InputDefaultField';
+import {Field, FormSpy, Form} from 'react-final-form';
+import {Mutation} from 'react-apollo';
+import {MUTATION_ADD_REVIEW} from '../../apollo/mutations';
+
+const SubmitReview = ({navigation, eventInfo}) => {
+  return (
+    <Mutation mutation={MUTATION_ADD_REVIEW}>
+      {(addReview, {loading, error, data}) => (
+        <>
+          <SafeAreaView>
```

Comment:
Remove this SafeAreaView to avoid style bugs.

Reply:
Complete.